README.md

```diff
@@ -1,359 +1,5 @@
-
-```bash
-# Overview of available commands
-nvm help
-
-# Install the latest stable version
-nvm install v4.4.3
-
-# Select the version to use
-nvm ls
-nvm use v4.4.3
-```
-
-* Install the nodejs dependencies
-
-```bash
-npm install
-```
-* Update css
-
-In case you need to update css, as gulp.js is used in this project, you would need to install it in case you haven't done that previously
-
-```bash
-npm install -g gulp
-```
-and execute sass task
-
-```bash
-gulp sass
-```
-
-# Pausing the daemon loop
-
-The daemon loop can be paused with:
-
-```bash
-sudo  wifi-connect stop
-```
-
-After this, the daemon loops and does nothing. In this state you may want to run "hidden" commands (see the sourcefor these), for example to execute functions for development and verification.
-
-Note: It is possible to execute commands that put the system into a non-working state. For example, bringing the AP UP/DOWN while wlan0 interface is managed by netork manager may result in an unworkable situation, possibly requiring reboot, or merely daemon restart.
-
-
-Restart the daemon normal loop cleanly with:
-
-
-```bash
-sudo  wifi-connect start
-```
-
-# Tests
-## Unit Tests
-
-You can run all tests by executing 
-```bash
-go test -v ./...
-```
-or
-```bash
-./run-check --unit
-```
-
-In order to run specific package test, you can:
-```bash
-go test -v ./<package>
-```
-
-To run a specific test:
-```bash
-go test -v -run <testname> 
-```
-
-For example:
-```bash
-go test -v ./wifiap
-go test -v ./wifiap -run TestShow
-```
-
-More info in https://golang.org/pkg/testing
-
-## Spread tests
-
-We have a set of spread (https://github.com/snapcore/spread) tests which
-are executed on a virtual machine
-
-In order to run those tests you need the follow things
-
-* ubuntu-image
-* spread
-
- You can install ubuntu-image snap
-
-```bash
-snap install --edge --classic ubuntu-image
-```
-
-NOTE: Because of confinement, snap spread cannot access virtualized environment. 
-For now, then, let's install spread from sources, this way:
-
-```bash
-# Note: here I use a temp folder as workdir but you can use 
-# any other path, temporary or permanent
-WORKDIR=`mktemp -d`
-export GOPATH=$WORKDIR
-go get -d -v github.com/snapcore/spread/...
-go build github.com/snapcore/spread/cmd/spread
-sudo cp spread /usr/local/bin
-```
-
-Make sure /usr/local/bin is in your path and is used as default:
-
-```bash
-which spread
-/usr/local/bin/spread
-```
-
-Now you have everything to run the test suite.
-
-```bash
-./run-tests
-```
-
-The script creates an image via ubuntu-image and makes it available
-to spread by copying it to ~/.spread/qemu or ~/snap/spread/<version>/.spread/qemu
-depending on if you're using a local spread version or the one from the
-snap.
-
-If you want to see more verbose debugging output of spread run
-
-```bash
-./run-tests --debug
-```
-
+This location is not been maintained anymore.
```

Comment:
Grammar. Please rewrite as: This project has moved, and this location is no longer maintained. Please see URL